eng/common/pipelines/templates/steps/bypass-local-dns.yml

```diff
@@ -3,4 +3,9 @@ steps:
     # https://github.com/actions/virtual-environments/issues/798
     - script: sudo ln -sf /run/systemd/resolve/resolv.conf /etc/resolv.conf
       displayName: Bypass local DNS server to workaround issue resolving cognitiveservices names
-      condition: and(succeededOrFailed(), eq(variables['OSVmImage'], 'ubuntu-18.04'))
\ No newline at end of file
+      condition: |
```

Comment:
@benbp did you intend these eng/common changes directly into the .NET repo? We should make sure these get into the tools repo.

Reply:
Ah! Missed the directory for this one. I'll make a sync PR.

Reply:
@weshaggard https://github.com/Azure/azure-sdk-tools/pull/1220